test/index.js

```diff
@@ -101,6 +101,15 @@ describe("depcheck", function () {
     });
   });
 
+  it("should report bad javascript", function testBadJS(done) {
+    var absolutePath = path.resolve("test/fake_modules/bad_js");
+
+    depcheck(absolutePath, {  }, function checked(unused) {
+      assert.notEqual(unused.invalidFiles, {});
```

Comment:
Why not merge this assertion into line 99 (which is commented out)?

Besides, could you add stronger assertion - use deepEqual to compare to `invalidFiles` object.